package.json

```diff
@@ -6,13 +6,13 @@
   "license": "MIT",
   "main": "lib/plugin.js",
   "scripts": {
+    "prebuild": "rimraf lib",
```

Comment:
I did a little cleanup of scripts using `pre`-hooks. Let me know if it is not ok and I'll revert